src/cookies.rs

```diff
@@ -0,0 +1,123 @@
+//! Cookie-related functionality for WebDriver.
+use serde::{Deserialize, Serialize};
+use time::OffsetDateTime;
+use webdriver::command::WebDriverCommand;
+
+use crate::client::Client;
+use crate::error;
+
+/// Type alias for a `cookie::Cookie`
+pub type Cookie<'a> = cookie::Cookie<'a>;
+
+/// JSON representation of a cookie as [defined by WebDriver](https://www.w3.org/TR/webdriver1/#cookies).
+#[derive(Debug, Deserialize, Serialize)]
+struct JsonCookie {
```

Comment:
It might make sense to expose this type directly instead of converting to/from a cookie::Cookie. The problem with using cookie::Cookie is that some fields will not be forwarded to WebDriver, even when set by the user (e.g., same_site).

Reply:
I'd actually specifically like to expose a type that isn't tied to the particular WebDriver standard so that the user doesn't have to work too hard to get to something that "just works" with the ecosystem, so let's keep this an implementation detail for the time being.

I also wonder if we should rename this `WebDriverCookie` instead, since that's what it actually represents — the fact that it's encoded as JSON is sort of neither here nor there.

Reply:
That makes sense!  I've renamed it to `WebDriverCookie`.